AdyenUI/UI/Form/Items/Value/Validatable/FormValidatableValueItemView.swift

```diff
@@ -13,102 +13,132 @@ open class FormValidatableValueItemView<ValueType, ItemType: FormValidatableValu
+
+    private func updateFooterDisplay(animated: Bool) {
+        if item.shouldShowValidationError {
+            displayError(item.validationFailureMessage, animated: animated)
+        } else {
+            displayHint(animated: animated)
         }
     }
     
-    // MARK: - Validation
+    // MARK: - Validation API
     
     public var isValid: Bool {
         item.isValid()
     }
     
+    // Called by form to trigger explicit validation (e.g., Pay button).
+    // Delegates to updateValidationStatus so subclasses can update their UI (e.g., border color).
     public func showValidation() {
         updateValidationStatus(forced: true)
     }
     
     open func updateValidationStatus(forced: Bool = false) {
         guard forced else {
-            showHint()
             accessibilityLabelView?.accessibilityLabel = item.title
             return
         }
-        if item.isValid() {
-            showHint()
-            accessibilityLabelView?.accessibilityLabel = item.title
-        } else {
-            showError(item.validationFailureMessage)
+
+        // Forced validation: update the single source of truth
+        // Observer will call updateFooterDisplay(animated: true) when this changes
+        // In the future, this logic moves to the Item and View listens to Item updates
+        item.shouldShowValidationError = !item.isValid()
+        updateAccessibility()
+        
+        triggerValidationErrorCallbackIfNeeded()
+    }
+
+    /// Clears validation error state.
+    internal func resetValidationStatus() {
+        item.shouldShowValidationError = false
+        updateFooterDisplay(animated: false) // Immediate feedback on focus
+        accessibilityLabelView?.accessibilityLabel = item.title
+    }
+
+    private func updateAccessibility() {
+        if item.shouldShowValidationError {
             accessibilityLabelView?.accessibilityLabel = [
                 item.title,
                 item.validationFailureMessage
             ].compactMap { $0 }.joined(separator: ", ")
+        } else {
+            accessibilityLabelView?.accessibilityLabel = item.title
         }
     }
-    
-    private func triggerValidationErrorIfNeeded() {
-        guard window != nil,
+
+    private func triggerValidationErrorCallbackIfNeeded() {
+        guard item.shouldShowValidationError,
+              window != nil,
               let validationStatus = item.validationStatus(),
-              let error = validationStatus.validationError else { return }
+              let error = validationStatus.validationError
+        else { return }
         item.onDidShowValidationError?(error)
     }
-    
-    internal func resetValidationStatus() {
-        showHint()
-        accessibilityLabelView?.accessibilityLabel = item.title
-    }
 
-    // MARK: - Footer Label (Hint/Error Display)
-    
-    package func showHint() {
+    // MARK: - Footer Display (Private)
+
+    private var footerAnimationKey: String { "footerLabel_visibility" }
```

Comment:
It can be a constant:
```swift
private let footerAnimationKey = "footerLabel_visibility"
```

Or even better to be moved into a `Constants` enum possibly